### PR TITLE
UITEN-260: prefer @folio/stripes exports to private paths when importing components

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
[UITEN-260](https://issues.folio.org/browse/UITEN-260)
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by Dmitriy-Litvinenko — APPROVED** [review]

File: src/settings/LocaleForm.js [hunk 1]
```diff
@@ -3,16 +3,14 @@ import PropTypes from 'prop-types';
 import { createIntl, createIntlCache, FormattedMessage, injectIntl } from 'react-intl';
 import { Field } from 'react-final-form';
 
-import {
-  Button,
+import { Button,
```
Comment (artem-blazhko) [inline src/settings/LocaleForm.js:6]:
```suggestion
import {
  Button,
```

File: src/settings/LocaleForm.js [hunk 2]
```diff
@@ -3,16 +3,14 @@ import PropTypes from 'prop-types';
...
   Col,
   CurrencySelect,
   Pane,
   PaneFooter,
   Row,
   Select,
-} from '@folio/stripes/components';
-import { TextLink } from '@folio/stripes-components';
+  TextLink } from '@folio/stripes/components';
```
Comment (artem-blazhko) [inline src/settings/LocaleForm.js:13]:
```suggestion
  TextLink,
  } from '@folio/stripes/components';
```

File: src/settings/LocationCampuses.js [hunk 3]
```diff
@@ -5,9 +5,11 @@ import {
   injectIntl,
 } from 'react-intl';
 import { ControlledVocab } from '@folio/stripes/smart-components';
-import { Select } from '@folio/stripes/components';
+import {
+  Select,
+  TextLink
```
Comment (OleksandrHladchenko1) [inline src/settings/LocationCampuses.js:10]:
```suggestion
  TextLink,
```

File: src/settings/LocaleForm.js (hunk 1 above)
Comment (UladzislauKutarkin) [reply src/settings/LocaleForm.js]: fixed.

File: src/settings/LocaleForm.js (hunk 2 above)
Comment (UladzislauKutarkin) [reply src/settings/LocaleForm.js]: fixed

File: src/settings/LocationCampuses.js (hunk 3 above)
Comment (UladzislauKutarkin) [reply src/settings/LocationCampuses.js]: fixed

**Review by OleksandrHladchenko1 — APPROVED** [review]

**Pull request merged**